framework/src/modules/chain/blocks/block.js

```diff
@@ -210,31 +209,34 @@ const create = ({
 	let size = 0;
 
 	const blockTransactions = [];
-	const payloadHash = crypto.createHash('sha256');
+	const transactionsBytesArray = [];
 
 	for (let i = 0; i < sortedTransactions.length; i++) {
 		const transaction = sortedTransactions[i];
-		const bytes = transaction.getBytes(transaction);
+		const transactionBytes = transaction.getBytes(transaction);
 
-		if (size + bytes.length > maxPayloadLength) {
+		if (size + transactionBytes.length > maxPayloadLength) {
 			break;
 		}
 
-		size += bytes.length;
+		size += transactionBytes.length;
 
 		totalFee = totalFee.plus(transaction.fee);
 		totalAmount = totalAmount.plus(transaction.amount);
 
 		blockTransactions.push(transaction);
-		payloadHash.update(bytes);
+		transactionsBytesArray.push(transactionBytes);
 	}
 
+	const transactionsBuffer = Buffer.concat(transactionsBytesArray);
```

Comment:
does it have same result as the previous implementation?
This is so much better than I was thinking =p

Reply:
It does :)